front/app/containers/BulkEditPage/BulkEditPage.tsx

```diff
@@ -239,6 +239,12 @@ class BulkEditPage extends React.PureComponent<BulkEditProps, BulkEditState> {
                     )
                 );
                 if (!labels.length) return null;
+                //Band-aid fix to the -99999999 breaking BucketsForLabelQuery, does not like the name field as -9999999999
+                labels.map((label, index)=>{
+                  if (label==-99999999999){
+                    labels[index]= "_missing"
```

Comment:
@bespinosa01 I'll look how it renders the 9999 in bulk but if it is showing "missing" - you can instead just skip displaying it all together until we find a proper fix in the future. It shouldn't be there as there is no real bulk action that coresponds to "updating" a set of trials in bulk to "missing or 09999 or whatever"

Reply:
@williamhoos let me know if you'd like me to remove it completely instead of where it's at